app/backend/wells/management/commands/legacy_records.py

```diff
@@ -0,0 +1,76 @@
+"""
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+import logging
+from timeit import default_timer as timer
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from wells.models import Well
+from wells.stack import StackWells
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Run from command line:
+    python manage.py legacy_records
+    """
+
+    def add_arguments(self, parser):
+        # Arguments added for debugging purposes.
+        # e.g. don't cleanup, don't upload: python manage.py export_databc --cleanup=0 --upload=0
```

Comment:
leftover comment?

Reply:
Yep! I thought I deleted this already... :thinking: